stonesoup/types/state.py

```diff
@@ -420,6 +420,8 @@ class TaggedWeightedGaussianState(WeightedGaussianState):
     """
     tag: str = Property(default=None, doc="Unique tag of the Gaussian State.")
 
+    BIRTH = 'birth'
```

Comment:
Just noted this hasn't appeared in the documentation. Adding something like this should work:
```suggestion
    BIRTH = 'birth'
    '''Tag value used to signify birth component'''
```

Reply:
Good catch, thanks!